### PR TITLE
Avoid running Danger for PRs coming from forks

### PR DESCRIPTION
**Pull request by MaxDesiatov**
Danger step always fails as `DANGER_GITHUB_API_TOKEN` is not available for such PRs.

### CODE REVIEW

**Comment by MaxDesiatov** [comment]
CC @jessesquires in follow-up to our discussion in https://github.com/SwiftWeekly/swiftweekly.github.io/pull/527

**Comment by jessesquires** [comment]
thanks @MaxDesiatov !

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -40,6 +40,7 @@ jobs:
         run: bundle exec jekyll build
 
       - name: danger
+        if: github.event.pull_request.head.repo.full_name == github.repository
```
Comment (jessesquires) [inline .github/workflows/ci.yml:43]: could we add a comment above this line explaining what/why ?

**Comment by jessesquires** [comment]
Looks good to me 👌 

Very unfortunate limitation though... 😞 I wish there was a way around this...

**Comment by jessesquires** [comment]
@MaxDesiatov -- also, I just added you to the org 😄 so you don't have to fork to contribute

**Comment by MaxDesiatov** [comment]
I think the only workaround is to set up a proper GitHub app, similar to what [Hound CI](https://houndci.com/) does, but they only run some old version of SwiftLint and don't support Danger. Interesting that no one have done this yet, maybe the Danger community considers something like this at some point.

**Comment by MaxDesiatov** [comment]
@jessesquires thanks for the org invitation, appreciated!

**Pull request merged**